frontend/test/metabase/scenarios/visualizations/pivot_tables.cy.spec.js

```diff
@@ -857,6 +857,56 @@ describe("scenarios > visualizations > pivot tables", () => {
 
     cy.findAllByText(/Totals for .*/i).should("have.length", 0);
   });
+
+  it.skip("should sort by metric (metabase#22872)", () => {
+    const questionDetails = {
+      dataset_query: {
+        database: SAMPLE_DB_ID,
+        query: {
+          "source-table": REVIEWS_ID,
+          aggregation: [["count"]],
+          breakout: [
+            ["field", REVIEWS.RATING, null],
+            ["field", REVIEWS.CREATED_AT, { "temporal-unit": "year" }],
+          ],
+        },
+        type: "query",
+      },
+      display: "pivot",
+    };
+
+    visitQuestionAdhoc(questionDetails);
+
+    cy.findByTextEnsureVisible("Created At: Year");
+    cy.findByTextEnsureVisible("Row totals");
+
+    assertTopMostRowTotalValue("149");
+
+    cy.icon("notebook").click();
+
+    cy.findByTextEnsureVisible("Sort").click();
+
+    popover()
+      .contains("Count")
+      .click();
+    cy.wait("@pivotDataset");
+
+    cy.button("Visualize").click();
+
+    assertTopMostRowTotalValue("23");
+
+    /**
+     * @param { string } value
+     */
+    function assertTopMostRowTotalValue(value) {
+      // Warning: Fragile selector!
+      // TODO: refactor once we have a better HTML structure for tables.
+      cy.get("[role=rowgroup] > div")
+        .eq(5)
```

Comment:
What is fragile about this? The only thing I can think of is that the table hasn't been rendered yet, but you could validate that with `cy.findByTextEnsureVisible("Row totals");` again before asserting `23`
Wouldn't it be the same problem if it was more HTML-ish? The optimal way would be with some DOM attributes.

Just an alternative pure CSS way of doing it.
```suggestion
      cy.get("[role=rowgroup] > div:nth-child(6)")
```
And could possibly avoid the invoke entirely with `[role=rowgroup] > div:nth-child(6) > div > div > div`, but that feels a little fragile.

Reply:
The fragile thing about this is that the test is tightly coupled with the **current** HTML implementation of the table. If we change it, the test would fail.

The selector itself is not problematic.